RESEARCH_A-Song-of-Ice-and-Fire-Aegons-Conquest-2026-06-02.md

```
@@ -0,0 +1,89 @@
+authority: LOGAN
+status: reference
+type: research-report
+topic: A Song of Ice and Fire — Aegon's Conquest
+verified-by: Claude Code (web research)
+date: 2026-06-02
+---
+
+# Aegon's Conquest
+
+> The war that fused weapon and bloodline into a throne: three dragonriders, fewer than 1,600 men, and two years that ended seven kingdoms and started a calendar. One of a trilogy with `[[RESEARCH_A-Song-of-Ice-and-Fire-The-Dragons-2026-06-02]]` (the weapon) and `[[RESEARCH_A-Song-of-Ice-and-Fire-House-Targaryen-2026-06-02]]` (the bloodline).
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)